src/get-started/timestamping-tutorial.md

```diff
@@ -398,26 +523,22 @@ signature, while `execute` performs the following operations:
 
 ```rust
 impl Transaction for TxTimestamp {
-    fn verify(&self) -> bool {
-        self.verify_signature(self.pub_key())
-    }
-
-    fn execute(&self, fork: &mut Fork) -> ExecutionResult {
-        let time = TimeSchema::new(&fork)
+    fn execute(&self, mut context: TransactionContext) -> ExecutionResult {
+        let tx_hash = context.tx_hash();
+        let time = TimeSchema::new(&context.fork())
```

Comment:
`TimeSchema` or `Schema`? Names are not consistent.

Reply:
@dvush `TimeSchema` is a type from time service. It's ok I guess.